polkadot/node/core/pvf/prepare-worker/src/lib.rs

```diff
@@ -103,17 +99,21 @@ fn send_response(stream: &mut UnixStream, result: PrepareResult) -> io::Result<(
 ///
 /// 1. Get the code and parameters for preparation from the host.
 ///
-/// 2. Start a memory tracker in a separate thread.
+/// 2. Start a new child process
 ///
-/// 3. Start the CPU time monitor loop and the actual preparation in two separate threads.
+/// 3. Start a memory tracker in a separate thread.
 ///
-/// 4. Wait on the two threads created in step 3.
+/// 3. Start the actual preparation in a separate thread.
+///
+/// 4. Wait on the thread created in step 3.
```

Comment:
Looks good, but step 3 appears twice. I would change to this:

```rs
/// 3. Start the memory tracker and the actual preparation in two separate threads.
///
/// 4. Wait on the two threads created in step 3.
```

The rest of the docstring is great!